src/registrar/forms/domain_request_wizard.py

```diff
@@ -647,20 +647,27 @@ class NoOtherContactsForm(BaseDeletableRegistrarForm):
 
 
 class CisaRepresentativeForm(BaseDeletableRegistrarForm):
+    cisa_representative_first_name = forms.CharField(
```

Comment:
Looks like according to the AC that these two fields are required (just wanted to let you know so you can fix the tests and the migration before going too far) 😅

`required=False` for `cisa_representative_first_name` and `cisa_representative_last_name`

Reply:
These fields are required by default.  The requirement is reinforced in the frontend (just verified on nl sandbox)

Reply:
AH just noticed it's already set to True in domain_information (and that this file is domain wizard, doh!)